python/Ganga/Lib/Localhost/LocalHostExec.py

```diff
@@ -88,14 +88,18 @@
 
 import time #datetime #disabled for python2.2 compatiblity
 
+if os.path.isfile(os.path.abspath(appscriptpath[0])):
```

Comment:
I'll drop it for now after reading comments in #418 but I'm under protest that using ```File``` and ```LocalFile``` is just going to cause un-needed confusion. (and frankly makes me suspect the string functionality here is something we shouldn't support)